### PR TITLE
Example.py: introduction of devices external list

### PR DESCRIPTION
**Pull request by chemelli74**

### CODE REVIEW

File: example.py [hunk 1]
```diff
@@ -39,10 +40,17 @@ async def cli():
 
 
 async def test_many():
-    device_options = [
-        aioshelly.ConnectionOptions("192.168.1.165", "admin", "test-password"),
-        aioshelly.ConnectionOptions("192.168.1.168"),
-    ]
+    with open("devices.json") as j:
+        json_data = json.load(j)
+    print(json_data)
```
Comment (balloob) [inline example.py:45]: unnecessary print.

File: devices.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{"devices": [{"ip": "192.168.1.1", "username": "admin", "password": "password"}, {"ip": "192.168.1.2"}, {"ip": "192.168.1.3"}]}
```
Comment (balloob) [inline devices.json:1]: I would much rather just have 1 device defined per line, and we load 1 line at a time into JSON.

File: example.py (hunk 1 above)
Comment (chemelli74) [reply example.py:45]: yup, leftover

File: example.py [hunk 3]
```diff
@@ -39,10 +40,16 @@ async def cli():
 
 
 async def test_many():
-    device_options = [
-        aioshelly.ConnectionOptions("192.168.1.165", "admin", "test-password"),
-        aioshelly.ConnectionOptions("192.168.1.168"),
-    ]
+    with open("devices.json") as j:
```
Comment (balloob) [inline example.py:43]: add this file to `.gitignore`

Comment (chemelli74) [reply example.py:43]: I though to have one in the repo as sample for users

Comment (balloob) [reply example.py]: We can do both. Add a default, then add it to gitignore to ignore future updates.

File: example.py [hunk 4]
```diff
@@ -43,10 +44,16 @@ def device_updated(device):
...
+    device_options = []
+    with open("devices.json") as fp:
+        for line in fp:
+            obj = json.loads(line)
+            device_data = []
+            if "username" in obj:
+                device_data = [obj["ip"], obj["username"], obj["password"]]
+            else:
+                device_data = [obj["ip"]]
+            device_options.append(aioshelly.ConnectionOptions(device_data[0]))
```
Comment (balloob) [inline example.py:56]: You're not using user/pass defined in `device_data`. There is also no reason to define `device_data` as intermediary. You can just read from `obj`.

**Review by balloob — APPROVED** [review]

**Pull request merged**